### PR TITLE
Release 1 0 4

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

**Review by MikeTaylor — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"> <modelVersion>4.0.0</modelVersion>
   <artifactId>mod-copycat</artifactId>
   <groupId>org.folio</groupId>
-  <version>1.0.4-SNAPSHOT</version>
+  <version>1.0.5-SNAPSHOT</version>
```
Comment (MikeTaylor) [inline pom.xml:4]: I assume that this is correct, and the back-end module release process skips right over the actual released version (1.0.5) somehow and goes straight into the next set of snapshots?

Comment (adamdickmeiss) [reply pom.xml:4]: Yes. The diff you see it the effect of tree commits that you see above, one of which is "[maven-release-plugin] prepare release v1.0.4" .. This on has 1.0.4 in it.

**Comment by adamdickmeiss** [comment]
Closing this commit as we'd like to have error message improvements be part of this release.

**Pull request closed without merging**